### ISSUE TITLE
Accessibility issue v12.2.0 - Media - Watermark image on page has no alt text

### ISSUE THREAD

**Issue by Truealex4** [umbraco/Umbraco-CMS#15068]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

v10.6.0 or v12.2.0

### Bug summary

When I run the lighthouse report on the media landing page, there is an issue reported where the watermark image that sits on the page explaining how to upload files is missing alt-text

### Specifics

/umbraco#/media
Not browser specific

### Steps to reproduce

1. Access umbraco v10.6.0. or v12.2.0
2. Login.
3. Select Media tab.
4. Run lighthouse accessibility report.

### Expected result / actual result

EXPECTED RESULT - No errors returned
ACTUAL RESULT - Error returned in lighthouse report as follwos:

![image](https://github.com/umbraco/Umbraco-CMS/assets/98405431/37f6c119-9d10-4261-8a20-eda1ad0c0e6b)
![image](https://github.com/umbraco/Umbraco-CMS/assets/98405431/7b7265fb-a8cb-4b89-961b-b0dfc2284a79)

**Comment by andr317c** [comment]
Thanks for posting this issue! I was able to reproduce this accessibility issue on versions 10.7.0 and 12.2.0. This is something we would like help with, I'll set this issue as up for grabs 😄
